ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -77,7 +77,7 @@ type Props = {
   autoplay: boolean,
   claimId: ?string,
   title: ?string,
-  channelName: ?string,
+  channelTitle: string,
```

Comment:
This changes the prop from `channelName` to `channelTitle`, but outside, `channelName` is still being determined and passed in.  Can clean that up since it is now an unused prop.

_Aside:  trying to recall why `channelName` was used for Chromecast ... was thinking either issues with the space character, or Title potentially being too long, but neither seem to make sense.  Never mind, lol._

Reply:
I've been using `channelName` for Chromecast in the Android app for a while now and it usually looks fine. The title can be too long, but it's rare and the character limit seems quite reasonable. I think in the long run we should get rid of the `channelName` everywhere including the chat. Using both the title and the name across the site can be a little bit confusing for the users.